.github/workflows/build.yml

```diff
@@ -181,6 +181,8 @@ jobs:
     env:
       CC: clang
       CXX: clang++
+      CFLAGS: -Wno-gnu-folding-constant
```

Comment:
Do we need to apply this flag not only in GitHub Workflow, but also in normal cmake builds?

Reply:
Yes. I'm going to use [this fix](https://github.com/openthread/openthread/pull/4944/files#diff-c1d434eb0e2a6df91b54b43e9ac19fc4) which applies this flag to one line of code only.